xray/core/plotting.py

```diff
@@ -0,0 +1,16 @@
+def _plot_dataarray(darray, *args, **kwargs):
+    """
+    Plot a DataArray
+    """
+    import matplotlib.pyplot as plt
+
```

Comment:
For 1D plotting, would it be possible to use the `pandas` plot functionality, rather than rewrite it here?